packages/backend/src/app/core/security/rate-limit.ts

```diff
@@ -0,0 +1,31 @@
+import RateLimitPlugin from '@fastify/rate-limit'
+import { FastifyPluginAsyncTypebox } from '@fastify/type-provider-typebox'
+import FastifyPlugin from 'fastify-plugin'
+import { Redis } from 'ioredis'
+import { createRedisClient } from '../../database/redis-connection'
+import { QueueMode, system } from '../../helper/system/system'
+import { SystemProp } from '../../helper/system/system-prop'
+
+export const rateLimitModule: FastifyPluginAsyncTypebox = FastifyPlugin(async (app) => {
+    await app.register(RateLimitPlugin, {
```

Comment:
I am a bit scared, especially about webhook endpoints. For example, Google servers sending us webhooks, or the engine querying for pieces information / store / connections.

On the other hand, we make a lot of requests from the frontend buit this is least worrying.

Reply:
disabled globally, enabled for authn only for the time being